src/CMDParse.cpp

```diff
@@ -0,0 +1,57 @@
+#include "CMDParse.h"
+
+EERAModel::ArgumentParser::ArgumentParser(int argc, char** argv)
+{
```

Comment:
Have you written any tests for this. Since you have control over the inputs and the outputs, this class would seem like an excellent opportunity for writing some effective tests.